### PR TITLE
[FONDY] Fondy API integration #2429

### PR DESCRIPTION
**Pull request by SofiiaBaziv**
integrated fondy api

### CODE REVIEW

**Review by AlekOK — APPROVED** [review]

File: src/app/main/component/ubs/components/ubs-personal-information/ubs-personal-information.component.ts [hunk 1]
```diff
@@ -159,7 +159,11 @@ export class UBSPersonalInformationComponent implements OnInit, OnDestroy {
       )
       .subscribe(() => this.findAllAddresses());
   }
-
+  getStringBetween(str, from, to): string {
+    return str.substring(
+      str.lastIndexOf(from) + from.length,
+      str.lastIndexOf(to));
+  }
```
Comment (JimHolman1199) [inline src/app/main/component/ubs/components/ubs-personal-information/ubs-personal-information.component.ts:166]: looks not really good 
what the point of this ? 
can we parse string to an array instead?

Comment (SofiiaBaziv) [reply src/app/main/component/ubs/components/ubs-personal-information/ubs-personal-information.component.ts:166]: the response from backend is a string with url. this is needed to find url in that string response

**Review by Edw1x — APPROVED** [review]

File: src/app/main/component/ubs/components/ubs-personal-information/ubs-personal-information.component.ts (hunk 1 above)
Comment (JimHolman1199) [reply src/app/main/component/ubs/components/ubs-personal-information/ubs-personal-information.component.ts]: ok !

**Comment by JimHolman1199** [comment]
let's wait until BE team come up with another solution about FONDY link
thanks

**Pull request merged**